test/IntegrationTests/MSTest.Acceptance.IntegrationTests/ServerModeTests.cs

```diff
@@ -40,7 +40,7 @@ public async Task DiscoverAndRun(string tfm)
         await Task.WhenAll(discoveryListener.WaitCompletion(), runListener.WaitCompletion());
         Assert.AreEqual(1, discoveryCollector.TestNodeUpdates.Count(x => x.Node.NodeType == "action"), $"Wrong number of discovery");
         Assert.AreEqual(2, runCollector.TestNodeUpdates.Count, $"Wrong number of updates");
-        Assert.IsFalse(logs.IsEmpty, $"Logs are empty");
+        Assert.IsTrue(logs.IsEmpty, $"Logs are not empty");
```

Comment:
Need to carefully double check if this is expected. And if yes, it's ideal to have another test that shows logs being not empty.